apps/nextra/public/scripts/install_cli.py

```diff
@@ -483,8 +483,62 @@ def get_target(self):
             self._write("")
             sys.exit(1)
 
-        # On Linux, we check what version of OpenSSL we're working with to figure out
-        # which binary to download.
+        if arch in ["arm64", "aarch64"]:
+            self._write(
+                colorize(
+                    "warning",
+                    "The given CPU architecture ({}) for Linux is in Beta, and only built for one version of Ubuntu-22.04.".format(arch),
+                )
+            )
+            return "Linux-aarch64"
+
+        # On Linux, determine the distribution and release
+        distro = "Unknown"
+        release = "Unknown"
+
+        try:
+          distro = subprocess.check_output(
+              ["lsb_release", "-is"],
+              universal_newlines=True,
+          )
+        except Exception:
+          self._write(
+            colorize(
+              "warning",
+              "Could not determine Linux distribution, assuming Ubuntu 22.04",
```

Comment:
It's better to test for the existence of `/etc/os-release`. If it doesn't exist, exit immediately because it's some weird unsupported system. If it exists, read it according to the [spec](https://www.freedesktop.org/software/systemd/man/latest/os-release.html).

Reply:
You can use this package: https://python-os-release.readthedocs.io/en/latest/

Reply:
Unfortunately that won't work.  This is supposed to be a one submit script.

Requiring other packages, requires first for someone to install python packages, and then submit it.

Reply:
Ok, then you can manually parse it. The syntax is KEY="<value>", one per line.